packages/jellyfish-network/src/BitcoinJsLib.ts

```diff
@@ -0,0 +1,40 @@
+import { getNetwork, NetworkName } from './Network'
+
+// eslint-disable-next-line @typescript-eslint/no-namespace
+export namespace BitcoinJsLib {
+  export interface Network {
+    messagePrefix: string
+    bech32: string
+    bip32: Bip32
+    pubKeyHash: number
+    scriptHash: number
+    wif: number
+  }
+
+  interface Bip32 {
+    public: number
+    private: number
+  }
+}
+
+/**
+ * https://github.com/bitcoinjs/bitcoinjs-lib
```

Comment:
```suggestion
 * https://github.com/bitcoinjs/bip32/blob/master/src/bip32.js#L18-L28
```
Shall we change to this to ease quick ref

Reply:
Needs to be a fully hash or else the Lines highlight might drift

Reply:
```suggestion
 * https://github.com/bitcoinjs/bip32/blob/ccc8fa7eedd7ff072f2f5c2d1ef89f0222391235/src/bip32.js#L18-L28
```